### ISSUE TITLE
IntegrityError occurs when deploy to Azure

### ISSUE THREAD

**Issue by lune94** [doccano/doccano#931]
Hi everyone

I would like to report an issue.

### System information
- **OS Platform and Distribution Azure Web App Linux container
- **Python version**: 3.6

### Describe the problem
We deployed Doccano APP via "Deploy to Azure" button on github. The application stopped running 12 days ago. The container cannot run anymore. Here is the webapp log:
2020-07-24T15:44:06.063Z INFO  - **docker run -d -p 9281:8000 --name doccanoapp_0_288079f1 -e PORT=8000 -e WEBSITES_ENABLE_APP_SERVICE_STORAGE=false -e WEBSITES_PORT=8000 -e WEBSITE_SITE_NAME=DoccanoApp -e WEBSITE_AUTH_ENABLED=False -e WEBSITE_ROLE_INSTANCE_ID=0 -e WEBSITE_HOSTNAME=*MYHOSTNAME* -e WEBSITE_INSTANCE_ID=*MY INSTANCE ID* doccano/doccano:latest**  
2020-07-24T15:44:33.539Z ERROR - Container doccanoapp_0_288079f1 for site doccanoapp has exited, failing site start
2020-07-24T15:44:33.549Z ERROR - Container doccanoapp_0_288079f1 didn't respond to HTTP pings on port: 8000, failing site start. See container logs for debugging.
2020-07-24T15:44:33.555Z INFO  - Stopping site doccanoapp because it failed during startup.

Here is the container log:
2020-07-24T00:01:33.529636296Z Operations to perform:
2020-07-24T00:01:33.529677899Z   Apply all migrations: admin, api, auth, authtoken, contenttypes, sessions, social_django
2020-07-24T00:01:33.529684799Z Running migrations:
2020-07-24T00:01:33.529689100Z   No migrations to apply.
2020-07-24T00:01:33.529693300Z   Your models have changes that are not yet reflected in a migration, and so won't be applied.
2020-07-24T00:01:33.529697700Z   Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
2020-07-24T00:01:42.125627948Z Traceback (most recent call last):
2020-07-24T00:01:42.125657550Z   File "/usr/local/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
2020-07-24T00:01:42.125663151Z     return self.cursor.execute(sql, params)
2020-07-24T00:01:42.125666851Z psycopg2.IntegrityError: duplicate key value violates unique constraint "auth_user_username_key"
2020-07-24T00:01:42.125670651Z DETAIL:  Key (username)=(MYLOGIN) already exists.

Thank you for your help

**Comment by br-dr3** [comment]
Just log is a little hard to guess, but did you tried to use the same docker volume to use docker-compose.dev.yml and docker-compose.prod.yml or something that could initialize database before running Deploy to Azure?

**Comment by JonnyFLDN** [comment]
I'm also having this issue with the Azure one click deployment. Anyone know where this potentially stems from?

**Comment by PATELS2R** [comment]
I faced similar issue on self deployment using `docker pull doccano/doccano` >> `docker container create ...` >> `docker container start doccano`

I have noticed that the container is accessible only once. If you close it (by accident or willfully) then you wont be able to start it again.

Following are lines from my docker logs might give a better insight
```
...
File "/usr/local/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 298, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.IntegrityError: UNIQUE constraint failed: auth_user.username

```
Seems like the `docker start` procedure is setting up the django database again (adding the admin credentials). But since we already have the entry of our admin credentials, it would throw integrity error for UNIQUE constrain in username parameter.
